### ISSUE TITLE
Find widget toggle buttons do not pass color contrast ratio

### ISSUE THREAD

**Issue by Shivanikupatkar** [microsoft/vscode#137441]
### GitHubTags
#A11yTCS; #A11ySev2; #A11yMAS; #A11yMAS #A11yWCAG; #DesktopApp; #Visual Studio Code Client;#BM-VisualStudioCode-Win32-Nov2021; #Color Contrast; #WCAG1.4.3; #Win32; #Win10; #AINotINScope;

### Environment Details
Application Name: Visual Studio Code Client
Version:1.62.1 (user setup)
Windows Version: Windows 10 (21H1)

### Repro Steps
1) Open Visual studio code editor    
2) Using keyboard navigation View Menu Bar
3) Select search option
4) Search and replace global content and check the color contrast ratio.

### Actual Result
 Color Contrast ratio of “Match case(Aa)” ,“Match whole word”, “Use regular expression”, “Preserve case” is 3.151:1which is less then required.

### Expected Result
 Color Contrast ratio of “Match case(Aa)” ,“Match whole word”, “Use regular expression”, “Preserve case” should be equal to 4.5:1  or greater than 4.5:1. 

### User Impact
Low vision users will face difficulty when they will navigate to the “Match case(Aa)” ,“Match whole word”, “Use regular expression”, “Preserve case” controls.

<img width="960" alt="A11y_Visual Studio Code Client_Search_ColorContrast  Color Contrast ratio of “match case(Aa)” ,“match whole word”, “use regular expression”, “preserve case” is 3 1511" src="https://user-images.githubusercontent.com/94344363/142564003-4fb1030c-575c-45a6-bcc8-bf1bf0c7102f.png">

**Comment by misolori** [comment]
@Shivanikupatkar these are icons so I believe the contrast ratio needs to pass 3.5:1. I'll also update this so it matches our hover background on the other toolbar icon:

<img width="107" alt="CleanShot 2021-11-18 at 20 29 01@2x" src="https://user-images.githubusercontent.com/35271042/142565028-35eef952-3664-4b25-a516-06e790d7c6d4.png">

**Comment by misolori** [comment]
I updated the icons to remove the opacity and they now appear at 3.9

<img width="402" alt="CleanShot 2021-11-19 at 10 49 19@2x" src="https://user-images.githubusercontent.com/35271042/142675665-a248b418-5b35-44f7-ad11-bb6431848c9d.png">

I also updated the find widget options to include hover feedback, just like our toolbar actions. There's a new color token for input options hover background, `inputOption.hoverBackground`, since they needed to be adjusted slightly to appear better on input backgrounds:

https://user-images.githubusercontent.com/35271042/142676212-8ebdd135-cace-4ca4-9cb0-5630a26021d7.mp4

For verification: verify the color contrast for search options passes 3:1 and that the new hover color token works and appears fine in all of our find widgets (search, find, terminal).
